releases/v21.2.0-beta.1.md

```diff
@@ -0,0 +1,1346 @@
+[#69070]: https://github.com/cockroachdb/cockroach/pull/69070
+[#69087]: https://github.com/cockroachdb/cockroach/pull/69087
+[#69091]: https://github.com/cockroachdb/cockroach/pull/69091
+[#69092]: https://github.com/cockroachdb/cockroach/pull/69092
+[#69106]: https://github.com/cockroachdb/cockroach/pull/69106
+[#69107]: https://github.com/cockroachdb/cockroach/pull/69107
+[#69113]: https://github.com/cockroachdb/cockroach/pull/69113
+[#69114]: https://github.com/cockroachdb/cockroach/pull/69114
+[#69122]: https://github.com/cockroachdb/cockroach/pull/69122
+[#69126]: https://github.com/cockroachdb/cockroach/pull/69126
+[#69152]: https://github.com/cockroachdb/cockroach/pull/69152
+[#69164]: https://github.com/cockroachdb/cockroach/pull/69164
+[#69173]: https://github.com/cockroachdb/cockroach/pull/69173
+[#69185]: https://github.com/cockroachdb/cockroach/pull/69185
+[#69202]: https://github.com/cockroachdb/cockroach/pull/69202
+[#69205]: https://github.com/cockroachdb/cockroach/pull/69205
+[#69224]: https://github.com/cockroachdb/cockroach/pull/69224
+[#69234]: https://github.com/cockroachdb/cockroach/pull/69234
+[#69238]: https://github.com/cockroachdb/cockroach/pull/69238
+[#69243]: https://github.com/cockroachdb/cockroach/pull/69243
+[#69251]: https://github.com/cockroachdb/cockroach/pull/69251
+[#69262]: https://github.com/cockroachdb/cockroach/pull/69262
+[#69273]: https://github.com/cockroachdb/cockroach/pull/69273
+[#69304]: https://github.com/cockroachdb/cockroach/pull/69304
+[#69311]: https://github.com/cockroachdb/cockroach/pull/69311
+[#69318]: https://github.com/cockroachdb/cockroach/pull/69318
+[#69320]: https://github.com/cockroachdb/cockroach/pull/69320
+[#69355]: https://github.com/cockroachdb/cockroach/pull/69355
+[#69370]: https://github.com/cockroachdb/cockroach/pull/69370
+[#69376]: https://github.com/cockroachdb/cockroach/pull/69376
+[#69377]: https://github.com/cockroachdb/cockroach/pull/69377
+[#69381]: https://github.com/cockroachdb/cockroach/pull/69381
+[#69388]: https://github.com/cockroachdb/cockroach/pull/69388
+[#69395]: https://github.com/cockroachdb/cockroach/pull/69395
+[#69405]: https://github.com/cockroachdb/cockroach/pull/69405
+[#69444]: https://github.com/cockroachdb/cockroach/pull/69444
+[#69457]: https://github.com/cockroachdb/cockroach/pull/69457
+[#69469]: https://github.com/cockroachdb/cockroach/pull/69469
+[#69470]: https://github.com/cockroachdb/cockroach/pull/69470
+[#69478]: https://github.com/cockroachdb/cockroach/pull/69478
+[#69480]: https://github.com/cockroachdb/cockroach/pull/69480
+[#69481]: https://github.com/cockroachdb/cockroach/pull/69481
+[#69483]: https://github.com/cockroachdb/cockroach/pull/69483
+[#69491]: https://github.com/cockroachdb/cockroach/pull/69491
+[#69502]: https://github.com/cockroachdb/cockroach/pull/69502
+[#69577]: https://github.com/cockroachdb/cockroach/pull/69577
+[0056828fd]: https://github.com/cockroachdb/cockroach/commit/0056828fd
```

Comment:
We can remove this line and all lines following. They are left-over links to commits that aren't linked to from this page.